datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/DatalakeDatabaseRestoreActions.java

```diff
@@ -120,32 +128,60 @@ protected Object getFailurePayload(DatalakeDatabaseRestoreCouldNotStartEvent pay
         };
     }
 
+    @Bean(name = "DATALAKE_FULL_RESTORE_IN_PROGRESS_STATE")
+    public Action<?, ?> datalakeFullResoreInProgress() {
+        return new AbstractSdxAction<>(SdxEvent.class) {
+
+            @Override
+            protected SdxContext createFlowContext(FlowParameters flowParameters, StateContext<FlowState, FlowEvent> stateContext, SdxEvent payload) {
+                return SdxContext.from(flowParameters, payload);
+            }
+
+            @Override
+            protected void doExecute(SdxContext context, SdxEvent payload, Map<Object, Object> variables) {
+                LOGGER.info("Full datalake restore is in progress for {} ", payload.getResourceId());
+                String operationId = (String) variables.get(OPERATION_ID);
+                sdxDatabaseDrService.updateDatabaseStatusEntry(operationId, SdxOperationStatus.SUCCEEDED, null);
+                sendEvent(context, DatalakeFullRestoreWaitRequest.from(context, operationId));
+
+                String restoreId = (String) variables.get(RESTORE_ID);
+                sdxDatabaseDrService.updateDatabaseStatusEntry(operationId, SdxOperationStatus.SUCCEEDED, null);
+                sendEvent(context, DatalakeFullRestoreWaitRequest.from(context, restoreId));
```

Comment:
we send 2 event from this action, is this intended?

Reply:
That is incorrect. I will fix it.